lib/config/fileconf.go

```diff
@@ -1453,6 +1453,16 @@ type SSH struct {
 	// DisableCreateHostUser disables automatic user provisioning on this
 	// SSH node.
 	DisableCreateHostUser bool `yaml:"disable_create_host_user,omitempty"`
+
+	// ForceListenInTunnel enables listening on the configured ListenAddress
+	// when connected to the cluster via a reverse tunnel. If no ListenAddress is
+	// configured, the default address is used.
+	//
+	// This allows the service to be connectable by users with direct network access.
+	// All connections still require a valid user certificate to be presented and will
+	// not permit any additional access. This is intended to provide an optional connection
+	// path to reduce latency if the Proxy is not co-located with the user and service.
+	ForceListenInTunnel bool `yaml:"force_listen_in_tunnel,omitempty"`
```

Comment:
The option name sounds really awkward because it could mean "listen in the tunnel" rather than "listen (externally) even if in tunnel mode" but I can't really come up with something that sounds better and isn't too verbose. `force_listen_in_tunnel_mode`? `force_listen_if_tunneling`? `force_tcp_listener_in_tunnel_mode`?

Reply:
`force_listen_and_tunnel`?

Reply:
+1 for `force_listen`. It's not an error to set the flag in direct mode right now, right?